pkg/rslog/logger_impl.go

```diff
@@ -214,6 +286,34 @@ var defaultLogger Logger
 var once = &sync.Once{}
 var mutex sync.RWMutex
 
+type buffer interface {
+	Buffer()
+}
+
+type flusher interface {
+	Flush()
+}
+
+// Buffer signals the default logger to buffer calls.
+// Returns a function that flushes the buffer.
```

Comment:
I think this comment isn't valid? since `Buffer` doesn't return anything

Reply:
My bad, it's from another implementation I was trying :man_facepalming: